x-pack/plugin/core/src/internalClusterTest/java/org/elasticsearch/snapshots/SourceOnlySnapshotIT.java

```diff
@@ -208,28 +210,20 @@ private void assertHits(String index, int numDocsExpected, boolean sourceHadDele
         }
     }
 
-    private IndexRequestBuilder[] snashotAndRestore(final String sourceIdx,
-                                                    final int numShards,
-                                                    final boolean minimal,
-                                                    final boolean requireRouting,
-                                                    final boolean useNested) throws InterruptedException, IOException {
+    private IndexRequestBuilder[] snapshotAndRestore(final String sourceIdx,
```

Comment:
`numShards` is always `1` and the `restore_minimal` setting isn't a thing anymore. It's just unused and quietly ignored since we don't do any repo setting validation.